README.md

```diff
@@ -4,17 +4,86 @@
 
 This repository contains code and tools for working with the Elastic APM AWS Lambda extension.
 
+Elastic APM Agents can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
+Currently the Node.js, Python and Java Agents are supported.
+
+The extension listens for data once per function invocation and forwards data to an APM Server.
```

Comment:
The list of supported agents is in the user docs - what do you think about just dropping these bits?

Reply:
Good point! Agree 👍